patch/files/scripts/outside.gd

```diff
@@ -777,11 +825,12 @@ var guildlocation
 func slaveguildslaves(location):
 	guildlocation = location
 	get_node("slavebuypanel").visible = true
-	var cost = 5
-	if globals.state.spec == 'Mage':
+	var cost = globals.expansionsettings.mindread_manacost*globals.expansionsettings.spellcost

```

Comment:
mindread_manacost no longer exists. the slave guild did not originally use the standard spell mana cost(3), the 5 cost is for all slaves in the guild, a discount if the player views more than one slave.

Reply:
Changed it to this.
ceil(globals.spelldict.mindread.manacost*globals.expansionsettings.spellcost)*1.5
Following that it was 5 before, I'll multiply the value by 1.5 after going through spellcost and the manacost for mindread, then ceiling it. Should represent it similarly.

Reply:
in the code you just posted the 1.5 is outside the ceil()

Reply:
whoops lol

Reply:
Fixed